index.html

```diff
@@ -5,11 +5,85 @@
-    <title>Document</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+        integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+    <link rel="shorcut icon" href="images/apple-brands.svg">
+    <title>Apple</title>
 </head>
 
 <body>
-    <h1>Good Luck for The new Project</h1>
+    <div class="container">
+        <header>
+            <div class="nav-wrapper">
+                <nav>
+                    <a class="white-color no-text-decoration " href="#"><i class="fab fa-apple fa-lg"></i></a>
+                    <a class="white-color no-text-decoration " href="#">Store</a>
+                    <a class="white-color no-text-decoration " href="#">Mac</a>
+                    <a class="white-color no-text-decoration " href="#">iPod</a>
+                    <a class="white-color no-text-decoration " href="#">iPhone</a>
+                    <a class="white-color no-text-decoration " href="#">iPad</a>
+                    <a class="white-color no-text-decoration " href="#">iTunes</a>
+                    <a class="white-color no-text-decoration " href="#">Support</a>
+                    <form class="parent-element">
+                        <input type="text" name="research-bar"><i class="fas fa-search white-color child-element"></i>
+                    </form>
+                </nav>
+            </div>
+            <div class="main-heading">
+                <h1 class="white-color">What will your verse be?</h1>
+                <h2 class="white-color">Everyone has something to share. See how people are using<br>
+                    iPad in their unique ways to add to the world's story.</h2>
+                <h3 class="parent-element white-color"><a class="no-text-decoration white-color" href="#">watch the
+                        "Your Verse" film and explore the story<i
+                            class="far fa-play-circle fa-lg child-element"></i></a></h3>
+            </div>
+        </header>
+        <main>
+            <div class="main-content">
+                <div class="main-content-box1 image-width">
+                    <a href="#"><img class="image-width" src="images/forward-thinking.jpg" alt="image of iPhone 5S"></a>
+                </div>
+                <div class="main-content-box2 image-width">
+                    <a href="#"><img class="image-width" src="images/for-the-color-full.jpg"
+                            alt="image of iPhone 5C"></a>
+                </div>
+                <div class="main-content-box3 image-width">
+                    <a href="#"><img class="image-width" src="images/HBD.jpg" alt="image of apple birthday"></a>
+                </div>
+                <div class="main-content-box4 image-width">
+                    <a href="#"><img class="image-width" src="images/supplier.jpg" alt="image of Apple Supplier"></a>
+                </div>
+            </div>
+        </main>
```

Comment:
- [ ] The four of them should have full-width. (If you remove the width on the css from the class _image-width_, the problem will be solved).

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/92503277-656b1280-f1d7-11ea-9704-10c0d14b9d6a.png) |![image](https://user-images.githubusercontent.com/61479648/92503295-6f8d1100-f1d7-11ea-986f-4c4b47aad310.png)  |